### PR TITLE
DSP HLE: Remove timing informations from ucodes

### PR DESCRIPTION
**Pull request by degasus**
On HLE, we don't emulate the timings on HLE, so there is also no need
to setup periods callbacks.

This fixes AX HLE when used for 48 kHz audio.

### CODE REVIEW

**Comment by delroth** [comment]
LGTM. I wish AX didn't rely on this update interval at all, to be fair :)

File: Source/Core/Core/HW/DSPHLE/UCodes/AX.cpp [hunk 1]
```diff
@@ -671,17 +671,13 @@ void AXUCode::Update()
 	}
 	else if (m_work_available)
 	{
+		m_work_available = false;
```
Comment (degasus) [inline Source/Core/Core/HW/DSPHLE/UCodes/AX.cpp]: This line is a bug since 4.0, maybe this should also be applied to stable?

**Comment by JMC47** [comment]
Fixes the missing FMV audio in AX-Wii games, Little King's Story, The Last Story, Donkey Kong Country Returns.  Fixes slow audio in many 48K games, including a bunch of random sports games I can't be bothered to type, Dave Mirra's Freestyle BMX 2 and some more sports games.

**Comment by JMC47** [comment]
Retested, LGTM

**Pull request merged**